tutor/specs/components/notes.spec.js

```diff
@@ -71,7 +71,7 @@ describe('Notes', () => {
   it('renders and matches snapshot', () => {
     notes.ux.isSummaryVisible = true;
     expect.snapshot(
-      <NotesWidget {...props} />,
+      <NotesWidget {...props}><p>hello</p></NotesWidget>,
```

Comment:
nah, that's just a spec change.  NotesWidget has a propType validation that requires a child.  Since the spec didn't provide one it was logging a warning which made the spec logs look nasty